interfaces/cython/cantera/onedim.py

```diff
@@ -215,6 +215,24 @@ def soret_enabled(self):
     def soret_enabled(self, enable):
         self.flame.soret_enabled = enable
 
+    @property
+    def flux_gradient_basis(self):
+        """
+        Get/Set whether or not species diffusive fluxes are computed with
+        respect to their mass fraction gradients ('mass')
+        or mole fraction gradients ('molar', default) when
+        using the mixture-avaraged transport model.
+        """
+        return self.flame.flux_gradient_basis
+
+    @flux_gradient_basis.setter
+    def flux_gradient_basis(self, basis):
+        if basis == 'molar' or basis == 'mass':
+            self.flame.flux_gradient_basis = basis
+        else:
+            raise ValueError("Valid choices are 'mass' or 'molar'."
+                             " Got {!r}.".format(basis))
```

Comment:
Checking the value of the flag here isn't really necessary -- the `_FlowBase.flux_gradient_basis` method can handle the error.